defaults/adv_settings.yml.default

```diff
@@ -22,9 +22,12 @@ traefik:
     jaeger: jaeger
   error_pages: no
 mounts:
-  remote: rclone_vfs
+  remote: rclone_vfs # or rclone_vfs_cache
   ipv4_only: no
   feeder: no
+  #rclone_vfs_cache_dir: "/mnt/cache/rclone/vfs"
```

Comment:
We have these variables tweakable using the inventory so they will not be placed here.